python/paddle/fluid/contrib/mixed_precision/bf16/amp_lists.py

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 
 import copy
+from .... import core
```

Comment:
IMHO following would be more readable and in case of this file directory change easier to maintain.
```suggestion
from paddle.fluid import core
```